python/cuspatial/cuspatial/core/geoseries.py

```diff
@@ -209,9 +215,9 @@ def part_offset(self):
         def point_indices(self):
             # Return a cupy.ndarray containing the index values from the
             # LineString GeoSeries that each individual point is member of.
-            offsets = self.part_offset.take(self.geometry_offset)
+            offsets = cp.array(self.part_offset)
             sizes = offsets[1:] - offsets[:-1]
-            return cp.repeat(self._series.index, sizes)
+            return cp.repeat(self._meta.input_types.index, sizes)
```

Comment:
I think this still needs to be updated, a regression relative to what has been merged lately.